modules/adkernelAdnBidAdapter.js

```diff
@@ -0,0 +1,145 @@
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+import { BANNER, VIDEO } from 'src/mediaTypes';
+
+const DEFAULT_ADKERNEL_DSP_DOMAIN = 'tag.adkernel.com';
+const VIDEO_TARGETING = ['mimes', 'protocols', 'api'];
+const DEFAULT_MIMES = ['video/mp4', 'video/webm', 'application/x-shockwave-flash', 'application/javascript'];
+const DEFAULT_PROTOCOLS = [2, 3, 5, 6];
+const DEFAULT_APIS = [1, 2];
+
+function isRtbDebugEnabled() {
+  return utils.getTopWindowLocation().href.indexOf('adk_debug=true') !== -1;
+}
+
+function buildImp(bidRequest) {
+  const sizes = bidRequest.sizes;
+  let imp = {
+    id: bidRequest.bidId,
+    tagid: bidRequest.placementCode
+  };
+  if (bidRequest.mediaType === 'video') {
```

Comment:
This should also check for the new `mediaTypes` property introduced in #1483 (with something like `|| (bidRequest.mediaTypes && bidRequest.mediaTypes.video)` or however you'd prefer) in case a pub defines a video adunit with `mediaTypes: {video: {}}` rather than `mediaType: 'video'`